formats/consultation/frontend/examples/closed_consultation.json

```diff
@@ -0,0 +1,70 @@
+{
+  "content_id": "1aa96aac-d978-4de2-b717-0f54acb33883",
+  "base_path": "/government/consultations/review-of-museums-in-england",
+  "description": "An online call for evidence to inform a review of the museums sector in England",
+  "public_updated_at": "2016-09-05T15:00:00.000+00:00",
+  "title": "Review of Museums in England",
+  "updated_at": "2016-09-05T15:00:00Z",
+  "schema_name": "consultation",
+  "document_type": "closed_consultation",
+  "locale": "en",
+  "details": {
+    "body": "<div class=\"govspeak\"><p>The Museums Review will consider how museums and galleries across England can thrive and become even more inclusive. The Review will look at how the sector operates, the challenges it faces and the role of government-sponsored museums.</p>\n\n<p>To help inform this review, DCMS would welcome your feedback through our online Calls for Evidence (see links in attached document).</p>\n\n<p>The Calls for Evidence are open until 31 October 2016.</p>\n\n<p>The full Terms of Reference for the review can be found in the above document.</p>\n\n<p>If you have any queries about the review, please contact DCMS at museums.review@culture.gov.uk</p>\n</div>",
+    "first_public_at": "2016-09-05T14:00:00+01:00",
+    "opening_date": "2016-09-05T14:00:00+01:00",
+    "closing_date": "2016-10-31T17:00:00+01:00",
+    "change_history": [
```

Comment:
Only `body`, `government` and `political` are required attributes of `details` so we shouldn't expect to see an empty `change_history`.